### DISCUSSION TITLE
Shortcuts for "copy the current location to the other window" and "copy the location of the other window into the current window" ?

### DISCUSSION THREAD

**Discussion by ultracello** [mikekazakov/nimble-commander#413] [Q&A] [answered]
Are shortcuts possible for:

- copy the current location to the other window

- copy the location of the other window into the current window

doable or if not, planned?

Many thanks!

Btw: super great software, many thanks! Not having Total Commander on macOS was most frustrations and is now done thanks nimble commander. It works very well.

**Comment by mikekazakov** [comment]
Thanks, as written in Help.md, F1/F2 has locations of other panels:
https://github.com/mikekazakov/nimble-commander/blob/main/Docs/Help.md#navigation

**Reply by ultracello** [reply, marked as answer]
Sorry, I think I was not clear, but no problem: CMD + Option + U was the thing I meant and the solution.
